dnf-behave-tests/features/steps/common/__init__.py

```diff
@@ -21,8 +21,8 @@
 from .file import file_timestamp
 from .module import get_modules_state
 from .repo import generate_metalink
-from .rpm import RPM
-from .rpm import diff_rpm_lists
+from .rpm_steps import RPM
```

Comment:
@pkratoch btw. sorry about the conflicts :frowning_face: 

I don't think you need to rename this, not entirely sure what is up with these relative imports though...? You can just use absolute import: `import steps.lib.rpm` should work with the new codebase?

Reply:
I needed this for the import of ``rpm`` (from the ``python3-rpm`` package) to expand the ``%{_libdir}`` macro, and since ``steps.rpm`` was imported as ``rpm``, there was a conflict. But it's not used anymore, so I can drop it.

Reply:
I dropped the commit that renamed ``rpm`` module to ``rpm_steps`` and resolved the conflicts.